SwiftVideoBackground.podspec

```diff
@@ -3,6 +3,7 @@ Pod::Spec.new do |s|
   s.version      = "3.0.0"
   s.summary      = "An easy to use Swift framework to play a video in the background of any UIView."
   s.description  = "SwiftVideoBackground is an easy to use Swift framework that provides the ability to play a video on any UIView. This provides a beautiful UI for login screens, or splash pages, as implemented by Spotify and many others"
+  s.screenshots  = "https://github.com/dingwilson/SwiftVideoBackground/raw/master/Assets/Spotify.gif"
```

Comment:
I did a `pod spec lint` and got 
```
- WARN  | screenshot: The screenshot https://github.com/dingwilson/SwiftVideoBackground/raw/master/Assets/Spotify.gif is not a valid image.
```

Reply:
I don't freaking understand why every link I try just downloads the gif and not displays it.

Reply:
Nice catch! Hmm thats weird, I'm using that same link rn for the README and it works just fine.. Maybe I have to change it to read from their raw.githubusercontent link.. I'll try that.

Reply:
Even if I use https://raw.githubusercontent.com/dingwilson/SwiftVideoBackground/master/Assets/Spotify.gif

it still fails to validate. What I noticed is that the other lib's gif directly plays in the browser, whereas mine seems to download... I can't seem to figure out why.

https://raw.githubusercontent.com/gmertk/GMStepper/master/Screenshots/screenshot_1.gif

Reply:
Yea...maybe just host the gif somewhere else as well?

Reply:
dang I just about to throw the gif onto my website, but then I realized thats run on GitHub too. 😂 Do you have any suggestions/preference for a good hosting site?

Reply:
How about https://imgur.com/

Reply:
Done. It passes pod spec lint now. 🎉

Reply:
I should probably add linting podspec as a check in travis..